### PR TITLE
[Table sanity check] sanity check for the table logged_in_users

### PR DESCRIPTION
**Pull request by akindyakov**
Fixes: #4928

### CODE REVIEW

File: osquery/tests/integration/tables/logged_in_users.cpp [hunk 1]
```diff
@@ -16,28 +16,19 @@
...
-  //      {"time", IntType}
-  //      {"pid", IntType}
-  //}
-  // 4. Perform validation
-  // validate_rows(data, row_map);
+TEST_F(LoggedInUsersTest, sanity) {
+  auto const rows = execute_query("select * from logged_in_users");
+  auto const row_map = ValidatatioMap{
+      {"type", NonEmptyString},
+      {"user", NormalType},
```
Comment (guliashvili) [inline osquery/tests/integration/tables/logged_in_users.cpp]: Can user be empty?

Comment (akindyakov) [reply osquery/tests/integration/tables/logged_in_users.cpp]: Unfortunately yes, e.g. users which type is `dead`.

File: osquery/tests/integration/tables/logged_in_users.cpp [hunk 2]
```diff
@@ -16,28 +16,19 @@
...
+      {"tty", NormalType},
+      {"host", NormalType},
+      {"time", NonNegativeInt},
```
[the remark this answers is not in the document]
Comment (akindyakov) [reply osquery/tests/integration/tables/logged_in_users.cpp]: Theoretically it can fail  on the system with wrong time setting. Not sure if it is ok :)

**Comment by akindyakov** [comment]
Thanks for review!

**Pull request merged**